test/user.test.js

```diff
@@ -0,0 +1,67 @@
+const { expect } = require('chai')
+const request = require('supertest')
+const app = require('../src/app')
+const User = require('../src/models/user')
+const {userOneId, userOne, setupDatabase} = require('./fixtures/db')
+
+beforeEach(setupDatabase)
```

Comment:
YOu only need to mock the controller function to return the response object that is suitable for you test.